### PR TITLE
Add support for additional market data bar types

### PR DESCRIPTION
**Pull request by rbasralian**

### CODE REVIEW

File: examples/example_beta_calc.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# API_PORT = 4001
+# API_PORT = 7496
+
+# API_PORT = 4002
+API_PORT = 7497
```
Comment (chipkent) [inline examples/example_beta_calc.py]: clean this up.  Do we need all of these commented out?

File: src/deephaven_ib/__init__.py [hunk 2]
```diff
@@ -132,14 +132,24 @@ class BarDataType(Enum):
...
+    FEE_RATE = 10
+    """Fee rate."""
+    YIELD_BID = 11
+    """Bid yield."""
+    YIELD_ASK = 12
+    """Ask yield."""
+    YIELD_BID_ASK = 13
+    """Bid/Ask yield."""
+    YIELD_LAST = 14
+    """Last yield."""
```
Comment (chipkent) [inline src/deephaven_ib/__init__.py:152]: Do any of these need to go in the exclusion list in `request_bars_realtime`?

**Review by chipkent — CHANGES_REQUESTED** [review]

File: src/deephaven_ib/__init__.py (hunk 2 above)
Comment (rbasralian) [reply src/deephaven_ib/__init__.py:152]: Don't believe so — think that only the existing ones (TRADES/MIDPOINT/BID/ASK) are [supported](https://interactivebrokers.github.io/tws-api/classIBApi_1_1EClient.html#a644a8d918f3108a3817e8672b9782e67)

File: examples/example_beta_calc.py (hunk 1 above)
Comment (rbasralian) [reply examples/example_beta_calc.py]: Updated the comment to clarify these — can still remove them if you'd prefer

**Comment by rbasralian** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by chipkent — APPROVED** [review]

**Pull request merged**